### PR TITLE
Vandana

### PR DESCRIPTION
**Pull request by van7b**

### CODE REVIEW

File: Resideo_eshopping/resideo_eshopping/lib/main.dart [hunk 1]
```diff
@@ -9,103 +10,10 @@ class MyApp extends StatelessWidget {
     return MaterialApp(
       title: 'Flutter Demo',
```
Comment (Roja-Vangapalli) [inline Resideo_eshopping/resideo_eshopping/lib/main.dart]: Change the title

File: Resideo_eshopping/resideo_eshopping/lib/models/products.dart [hunk 2]
```diff
@@ -0,0 +1,26 @@
+class Products{
+  final String category;
+  final String image;
+  final int inventory;
+  final String longDescription;
```
Comment (Roja-Vangapalli) [inline Resideo_eshopping/resideo_eshopping/lib/models/products.dart]: variables should not be declared as final in models

File: Resideo_eshopping/resideo_eshopping/lib/pages/productsListPage.dart [hunk 3]
```diff
@@ -0,0 +1,112 @@
+import 'package:flutter/material.dart';
+import 'package:resideo_eshopping/models/products.dart';
+import 'package:resideo_eshopping/repository/products_repository.dart';
+import 'package:resideo_eshopping/widgets/products_tile.dart';
+
+class ProductsListPage extends StatefulWidget {
+  ProductsListPage({Key key, this.title}) : super(key: key);
+
+  final String title;
```
Comment (Roja-Vangapalli) [inline Resideo_eshopping/resideo_eshopping/lib/pages/productsListPage.dart]: remove final for all the variables where it is not required

File: Resideo_eshopping/resideo_eshopping/lib/main.dart (hunk 1 above)
Comment (van7b) [reply Resideo_eshopping/resideo_eshopping/lib/main.dart]: Changed the title from 'Flutter Demo' to 'Resideo e-Shopping'.

**Pull request merged**